src/test/groovy/org/mockito/release/notes/vcs/DefaultCommitApproverTest.groovy

```diff
@@ -0,0 +1,52 @@
+package org.mockito.release.notes.vcs
+
+import spock.lang.Specification
+
+class DefaultCommitApproverTest extends Specification {
+
+    def "should approve commit with sample message"() {
+        def approver = new DefaultCommitApprover()
+        def commit = new GitCommit("sampleId", "sample@email.com", "sampleAuthor", "sample commit message")
+
+        when:
+        def result = approver.isTrue(commit)
+
+        then:
+        result == true
+    }
+
+    def "should decline commit with [ci skip] text in message"() {
+        def approver = new DefaultCommitApprover()
+        def commit = new GitCommit("sampleId", "sample@email.com", "sampleAuthor", "commit [ci skip] message")
+
+        when:
+        def result = approver.isTrue(commit)
+
+        then:
+        result == false
+    }
+
+    def "should decline commit with defined postfix in message"() {
+        def samplePostfix = "samplePostfix"
```

Comment:
True but in this case postfix is both a parameter and part of the message. It is not clear which of that you mean in the test name. And in the next test it is clear